docs/.vuepress/config.js

```diff
@@ -26,6 +26,40 @@ module.exports = {
         after: '</div>'
       },
     ],
+    [
+      require('../../plugins/faqs'),
+      {
+        // Where is the FAQ data stored, relative to the root directory.
+        faqsBase: './faqs',
+        // The main repository, used to build the "Edit on Github" links.
+        githubRepo: 'https://github.com/dobromir-hristov/vuecommunity',
```

Comment:
coudlnt we use the `themeConfig.repo` ? Just asking, its not such a big deal.

Reply:
I didn't notice that! I'll check if it's possible now.

Reply:
It's not possible, however we can define a constant at the top of the file and use it in both locations?